handout/features/es6_modules.md

```diff
@@ -1,8 +1,33 @@
-# ES6 Modules
+# ECMAScript 2015 Modules
 
-ES6 also introduces the concept of a module, which works similar to other languages. Defining an ES6 module is quite easy: each file is assumed to define a module and we specify its exported values using the `export` keyword. 
 
-Loading ES6 modules is a little trickier. In an ES6-compliant browser you use the `System` keyword to load modules asynchronously. To make our code work with current browsers, however, we will use the SystemJS library as a polyfill:
+ECMAScript 2015 introduced _module_ support. A module in ES2015 is single file that allows code and data to be isolated, it helps in organizing and grouping code logically. In other languages it's called a package or library.
+
+All code and data inside the module has file scope, what this means is they are not accessible from code outside the module. To share code or data outside a module, it needs to be exported using the **export** keyword.
+
+```js
+// File: circle.js
+
+export const pi = 3.141592;
+
+export function circumference(diameter) {
```

Comment:
Better to use `export const circumference = diameter => diameter * pi` since this section is about ES6